craft_providers/actions/snap_installer.py

```diff
@@ -139,6 +190,12 @@ def inject_from_host(*, executor: Executor, snap_name: str, classic: bool) -> No
             check=True,
             capture_output=True,
         )
+
+        InstanceConfiguration.update(
```

Comment:
Is this `InstanceConfiguration.update` call expected to raise `subprocess.CalledProcessError`? If not, it should be outside the "try block"...